property/views.py

```diff
@@ -147,6 +146,29 @@ class CreateAndListPropertyView(generics.ListCreateAPIView):
     filter_class = PropertyFilter
     parser_classes = (MultiPartParser, FormParser)
 
+    @staticmethod
+    def validate_building_rooms(data):
+        if not data.get('property_type') or data.get('property_type') == 'B':
```

Comment:
Nice work @wangonya , what do you think of having this as your `validate_building_rooms`, 

```py
    @staticmethod
    def validate_building_rooms(data):
        if data.get('propety_type') == 'B':
            if data.get('bathrooms') == ('0' or None):
                return False
            if data.get('bedrooms') == ('0' or None):
                return False
         else return False
```

My thoughts are, it gets rid of all the calls to `.get` and its a bit more readable

Reply:
Thanks for the feedback @Tevinthuku. Yeah, that does look better. Let's sync up and try it out.